src/Opdex.Platform.Infrastructure/PlatformInfrastructureMapperProfile.cs

```diff
@@ -426,7 +426,7 @@ public PlatformInfrastructureMapperProfile()
             .ForMember(dest => dest.Symbol, opt => opt.MapFrom(src => src.Symbol))
             .ForMember(dest => dest.Decimals, opt => opt.MapFrom(src => src.Decimals))
             .ForMember(dest => dest.Sats, opt => opt.MapFrom(src => src.Sats))
-            .ForMember(dest => dest.TotalSupply, opt => opt.MapFrom(src => src.TotalSupply))
+            .ForMember(dest => dest.TokenSummary, opt => opt.MapFrom(src => src.TokenSummary))
             .ForMember(dest => dest.CreatedBlock, opt => opt.MapFrom(src => src.CreatedBlock))
```

Comment:
This we want to keep total supply mapping here

Reply:
Re-added